README.md

```diff
@@ -45,7 +45,8 @@ run `mix guardian.db.gen.migration` to generate a migration.
 To sweep expired tokens from your db you should add `Guardian.DB.Token.SweeperServer` to your supervision tree.
 
 ```elixir
-  worker(Guardian.DB.Token.SweeperServer, [])
+  %{id: :guardian_db_token_sweeper, 
```

Comment:
@Immortalin it looks like there's an even simpler option for this and we should probably use that: https://hexdocs.pm/elixir/Supervisor.html#module-child_spec-1

```suggestion
children = [
  {Guardian.DB.Token.SweeperServer, []}
]
```